doc/tutorials/acrn-secure-boot-with-grub.rst

```diff
@@ -5,16 +5,16 @@ Enable ACRN Secure Boot With GRUB
 
 This document shows how to enable ACRN secure boot with GRUB including:
 
--  ACRN Secure Boot Sequence
--  Generate GPG Key
--  Setup Standalone GRUB EFI Binary
--  Enable UEFI Secure Boot
+-  `ACRN Secure Boot Sequence`_
+-  `Generate GPG Key`_
+-  `Setup Standalone GRUB EFI Binary`_
+-  `Enable UEFI Secure Boot`_
 
 **Validation Environment:**
 
 - Hardware Platform: TGL-I7, Supported hardware described in
```

Comment:
@NanlinXie  Could we mention this platform instead of this generic `TGL-I7` that is not on the list: `NUC11TNHi5`?

Reply:
Shall we change to "TGL"?

Reply:
Per offline discussion with @NanlinXie @dbkinder, I will change it to "TGL" for 2.7 release.